package.json

```diff
@@ -68,7 +74,5 @@
   "nx": {
     "includedScripts": []
   },
-  "workspaces": [
-    "nodes/*"
-  ]
+  "packageManager": "pnpm@9.4.0"
```

Comment:
What is the benefit of providing this information? Is it necessary to pin it to this specific version, would it not be sufficient to pin it only to the major version?

Reply:
The reason: https://github.com/pnpm/action-setup/tree/v4/?tab=readme-ov-file#version

But we can also just pin it to the major version.

Reply:
Nice catch, the documentation seems to be incorrect, the action works flawlessly in the test repository without providing this information to the action or the package.json. 

But sure we should follow the documentation and I would say we pin it at least to the major version.

Reply:
That's bad, I hope they will improve the documentation from the action so it reflects this requirement. The Node.js documentation is not very detailed about this property due to its experimental state.